eurocropsml/dataset/preprocess.py

```diff
@@ -15,11 +16,15 @@
 from tqdm import tqdm
 
 from eurocropsml.dataset.config import EuroCropsDatasetPreprocessConfig
-from eurocropsml.utils import _unzip_file
 
 logger = logging.getLogger(__name__)
 
 
+def _unzip_file(zip_filepath: Path, extract_to_path: Path) -> None:
+    with zipfile.ZipFile(zip_filepath, "r") as zip_ref:
+        zip_ref.extractall(extract_to_path)
+
+
 def download_dataset(preprocess_config: EuroCropsDatasetPreprocessConfig) -> None:
```

Comment:
Once we have the final Sentinel-1 data, we should upload them to Zenodo and update this function with the possibility to select whether one wants to download S1 and S2 or only one of the two. I think this should be done before merging any of the current changes.